diff --git git-2.17.1.orig/contrib/credential/wincred/Makefile git-2.17.1/contrib/credential/wincred/Makefile
index 6e992c0..61b5792 100644
--- git-2.17.1.orig/contrib/credential/wincred/Makefile
+++ git-2.17.1/contrib/credential/wincred/Makefile
@@ -16,7 +16,7 @@ git-credential-wincred.exe : git-credential-wincred.c
 	$(LINK.c) $^ $(LOADLIBES) $(LDLIBS) -o $@
 
 install: git-credential-wincred.exe
-	$(INSTALL) -m 755 $^ $(libexecdir)
+	$(INSTALL) -m 755 $^ $(DESTDIR)$(libexecdir)
 
 clean:
 	$(RM) git-credential-wincred.exe
diff --git git-2.17.1.orig/contrib/credential/wincred/git-credential-wincred.c git-2.17.1/contrib/credential/wincred/git-credential-wincred.c
index 86518cd..49b85f8 100644
--- git-2.17.1.orig/contrib/credential/wincred/git-credential-wincred.c
+++ git-2.17.1/contrib/credential/wincred/git-credential-wincred.c
@@ -6,6 +6,7 @@
 #include <stdio.h>
 #include <io.h>
 #include <fcntl.h>
+#include <wchar.h>
 
 /* common helpers */
 
@@ -290,8 +291,8 @@ int main(int argc, char *argv[])
 		die(usage);
 
 	/* git use binary pipes to avoid CRLF-issues */
-	_setmode(_fileno(stdin), _O_BINARY);
-	_setmode(_fileno(stdout), _O_BINARY);
+	_setmode(fileno(stdin), O_BINARY);
+	_setmode(fileno(stdout), O_BINARY);
 
 	read_credential();
 
